joint_limits/include/joint_limits/data_structures.hpp

```diff
@@ -21,9 +21,21 @@
 #include <optional>
 #include <string>
 
+#define DEFINE_LIMIT_STRUCT(limit) \
+  struct limit                     \
+  {                                \
+    double lower_limit;            \
+    double upper_limit;            \
+  };
```

Comment:
```suggestion
#define DEFINE_LIMIT_STRUCT(LimitType) \
  struct LimitType                     \
  {                                    \
    double lower_limit;                \
    double upper_limit;                \
  };
```

Let's also rename the arg to `LimitType`